cmd/crank/xpkg/build.go

```diff
@@ -21,9 +21,12 @@ import (
 	"path/filepath"
 
 	"github.com/google/go-containerregistry/pkg/name"
+	v1 "github.com/google/go-containerregistry/pkg/v1"
```

Comment:
I’m pretty new in the go-game, so this might be a stupid question, but why would you need `v1` here? Isn’t this the default already?

Reply:
yeah, it's not required indeed, it looks like IDEs like adding that in for some reason, I removed that in a few other places too. Thanks for raising the question! 🙏